### ISSUE TITLE
GUI: touch pad scrolling

### ISSUE THREAD

**Issue by libbyhopfauf** [mipops/dvrescue#433]
In the GUI, for any part where scrolling is available: using touch pad motions to scroll (vs. clicking and holding) works for up/down, but not for left/right.

**Comment by dericed** [comment]
This applies in particular to the frame table, segment table and file table.

**Comment by dericed** [comment]
still an issue. Help us @ElderOrb :D

**Comment by ElderOrb** [comment]
as I can see - the only place where we have horizontal scrollbar - is this table: 

![image](https://user-images.githubusercontent.com/23638/178143819-58600adc-dd02-4772-8408-66209abd5e2f.png)

frame table & segment table doesn't seem to even have horizontal scrolling capabilities atm. 

do you want to add it here?

also not sure about 'zoom in/out', any examples of where we have such a functionality?

@dericed @libbyhopfauf ^^

**Comment by dericed** [comment]
@ElderOrb yes i think that's right that the frame table and segment table could benefit from having horizontal scroll capabilities. Currently I can move vertically in these tables via the touchpad but not horizontally.

For zoom in/out, I presume this references a two-finger pinch or unpinch gesture on the touchpad, which could be relevant for components with the zoom in/out option (the graph)

**Comment by libbyhopfauf** [comment]
@ElderOrb @dericed I agree! That would horizontal scroll capabilities would be super helpful!

**Comment by libbyhopfauf** [comment]
I just wanted to jump back in here and note that this is still an issue. Also, on a related note, is there a way to make it so that the scroll bars are always present (vs. only appearing when you hover your cursor over the area)? I've heard from a few users that this is not intuitive.
